src/pages/sponsor.module.css

```diff
@@ -0,0 +1,77 @@
+.container {
+  margin: 0 auto;
+  padding: 24px 24px;
+  max-width: 960px;
+  background-color: #121212;
+  color: white;
+  align-items: center;
+}
+
+.dispImage {
+  object-fit: contain;
+  object-position: center;
+  width: 100%;
+  max-width: 500px;
+  height: auto;
+}
+
+.h2alt {
+  margin-top: 20px;
+}
+
+.imageContainerLeft {
+  margin-top: 0;
+  margin-left: 30px;
+  display: flex;
+  flex: 0 0 33%;
+}
+
+.imageContainerRight {
+  margin-top: 20px;
+  margin-right: 30px;
+  display: flex;
+  flex: 0 0 37%;
+}
+
+.rowText {
+  display: flex;
+  flex-direction: row;
+}
+
+.section {
+  background-color: #262626;
+  color: white;
+}
+
+@media only screen and (max-width: 768px) {
+  .h2alt, h2 {
+    text-align: center;
```

Comment:
This is different to the index page (which keeps headings left-aligned). We should revisit this later to ensure consistency across pages.